src/vs/workbench/contrib/notebook/browser/contrib/editorStatusBar/editorStatusBar.ts

```diff
@@ -305,9 +305,9 @@ export class NotebookIndentationStatus extends Disposable implements IWorkbenchC
 		}
 
 		const cellEditorOverridesRaw = editor.notebookOptions.getDisplayOptions().editorOptionsCustomizations;
-		const indentSize = cellEditorOverridesRaw['editor.indentSize'] ?? cellOptions?.indentSize;
```

Comment:
It is possible for `cellEditorOverridesRaw` to be undefined
Thus causing this code path to fall over.